Automattic/jetpack
# Build tools: Remove carriage returns in build-module-headings-translations.php

#### Changes proposed in this Pull Request:


I went nuclear and removes all of the carriage returns in this script. You should no longer see `^M` carriage returns after building normally or running this script. 

#### Testing instructions:
- `gulp php:module-headings`
- You should no longer see `build-module-headings-translations.php` has changed, nor has any of the `^M`s

#### Proposed changelog entry for your changes:


*none needed
